pql/ast.go

```diff
@@ -112,6 +114,15 @@ func (q *Query) addField(field string) {
 	}
 }
 
+// validateArgField ensures that field does not already
+// exist as a key in the Args map before adding the new
+// key/value.
+func (q *Query) validateArgField(elem *callStackElem) {
+	if _, exists := elem.call.Args[elem.lastField]; exists {
+		panic(fmt.Sprintf("multiple instances of argument '%s' provided", elem.lastField))
```

Comment:
I forget... does a panic in the parser then manifest itself as an error?

Reply:
we don't want to panic though, do we? I'd think we would want to return an informative error to the user who issued the query with a duplicate field

Reply:
I agree, but is seems everything in `pql/ast.go` is handled with panics on malformed inputs.

Doing proper errors will require some refactoring. For this issue in particular, because the arguments are stored in a map, this is the only place where we can tell that a duplicate argument was provided. Effectively, it's "last one wins" because the map entry gets overwritten.

I think we could either introduce error handling here, or we could use a different data structure for the arguments (i.e. one that would allow duplicate keys).